.changeset/chilled-buckets-sneeze.md

```diff
@@ -0,0 +1,6 @@
+---
+'@firebase/firestore': patch
+'firebase': patch
+---
+
+Check navigator.userAgent, in addition to navigator.appVersion, when determining whether to work around an IndexedDb bug in Safari. Add check for Mobile/# in addition to Version/# to include WKWebView safari versions in this check.
```

Comment:
Please remove the 2nd sentence from the changeset. The first sentence is the right amount of information for the release notes. If they want more information they can click on the link from the release notes and see your full change.